libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/LeveledLogger.scala

```diff
@@ -82,10 +92,11 @@ private[logging] sealed abstract class LeveledLogger {
   protected def log(msg: String, t: Throwable): Unit
   protected def log(m: Marker, msg: String, t: Throwable): Unit
   protected def log(fmt: String, arg: AnyRef): Unit
+  protected def log(m: Marker, msg: String): Unit
 
   final def apply(msg: => String)(implicit loggingContext: LoggingContext): Unit =
     if (isEnabled)
-      loggingContext.ifEmpty(log(msg))(log(s"$msg (context: {})", _))
+      loggingContext.ifEmpty(log(msg))(log(_, msg))
```

Comment:
This would mean that the logging context is completely removed unless you are using JSON logging. There are usages where logging a meaningful message relies on the context being printed along with the message for it to be meaningful (in those cases the message only prints what even happened, e.g. something along the lines of "Request received", but the data about the request is in the context -- this is a deliberate choice to avoid the same information to appear in both the message and the context). I think it's important to find a way to print the context along with the message is a non-structured encoder is chosen.

Reply:
with the adjusted logback.xml file this should be now fine.